src/bepasty/app.py

```diff
@@ -69,7 +69,7 @@ def setup_secret_key(app):
 def create_app():
     app = Flask(__name__)
 
-    app.config.from_object('bepasty.config.Config')
+    app.config.from_object('bepasty.config')
     if os.environ.get('BEPASTY_CONFIG'):
         app.config.from_envvar('BEPASTY_CONFIG')
```

Comment:
btw, did you practically try whether overriding a default config value (residing in `bepasty.config`) via a custom config module given by `BEPASTY_CONFIG` actually works correctly?

Reply:
Yep, just re-tested and I can see my changes on the site.

``` bash
cat bepasty.conf && BEPASTY_CONF=$(pwd)/bepasty.conf bepasty-server
# SITENAME = 'bepasty.local'
# DEFAULT_PERMISSIONS = 'admin,list,create,modify,read,delete'
#  * Starting bepasty server...
#  * Serving Flask app 'bepasty.app' (lazy loading)
#  * Environment: production
#    WARNING: This is a development server. Do not use it in a production deployment.
#   Use a production WSGI server instead.
#  * Debug mode: off
#  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
```

Reply:
ok, great!